onnxconverter_common/float16.py

```diff
@@ -388,3 +389,104 @@ def sort_topology(graph_proto):
             for g in attr.graphs:
                 if isinstance(g, onnx_proto.GraphProto):
                     sort_topology(g)  # sort sub-graph
+
+
+def remove_unnecessary_cast_node(graph_proto):
```

Comment:
Is there a way to create a test to cover the issue this function resolves?

Reply:
The test case failed so have to change this code.